docs/src/documentation/05-development/04-migration-guides/v10.mdx

```diff
@@ -0,0 +1,101 @@
+redirect_from:
+  - /migration-guides/v10/
+---
+
+# Orbit Migration Guide v10
+
+This migration guide focuses on the process of migrating from Orbit v9.1 to v10.0, as some breaking changes were introduced.
+With this guide we aim to walkthrough all the breaking changes and how they can be addressed, allowing the migration to be smoother and effortless.
+
+This version of Orbit includes the first batch of components styled with Tailwind, instead of Styled Components. Note that you can keep using Styled Components, especially on your custom components. But we recommend start migrating them to Tailwind as well.
+
+Components in Tailwind should have the same API as before, so no breaking changes are expected in that regard. However, and since some of the migrated components went through a deep refactoring, we recommend being on the lookout for unexpected and undocumented changes. Those should be reported so that we can fix them.
+
+## Breaking changes
+
+### Tailwind requirement
+
+As mentioned before, this version of Orbit includes the first batch of components styled with Tailwind. Therefore, Tailwind must be installed and configured in your project for you to be able to use these components.
+Please refer to the [official Tailwind documentation](https://tailwindcss.com/docs/installation) and the [Orbit's Tailwind setup documentation](/foundation/tailwind/installation) for more information on how to do that.
+
+As a summary, you should have Tailwind installed and configured in your project, as well as the Orbit preset for Tailwind included on the config file.
+
+### StyledBox removed
+
+The `StyledBox` component is no longer exported. This allowed custom Styled Components to target the `Box` components from Orbit and apply styles to it. Therefore, things like the following code will not work anymore:
+
+```jsx
+import { StyledBox } from "@kiwicom/orbit-components";
+
+const SomeWrapper = styled.div`
+  height: 100%;
+
+  ${StyledBox} {
+    padding: 10px;
+  }
+`;
+```
+
+This could be used to add custom styles to the `Box` component, one of the exceptions we have in Orbit Design System.
+Instead of using `StyledBox`, you can now target the components by using the `.orbit-box` selector:
+
+```jsx
+const SomeWrapper = styled.div`
+  height: 100%;
+
+  .orbit-box {
+    padding: 10px;
+  }
+`;
+```
+
+### Align prop in Text
+
+The `Text` component features an `align` prop that directly matches the `text-align` CSS property. This property used to have a default value of `"left"` and Styled Components was automatically dealing with RTL and converting `"left"` to `"right"` when the browser was presenting a page in RTL.
+
+Now, this is no longer the case. Therefore, we have changed the default value to be `"start"` so that it keeps the same behavior as before. However, if you were using the `align` prop with the value `"left"` or `"right"` and relying on an automatic RTL convertion, you should now use `"start"` instead.
```

Comment:
```suggestion
Now, this is no longer the case. Therefore, we have changed the default value to be `"start"` so that it keeps the same behavior as before. However, if you were using the `align` prop with the value `"left"` or `"right"` and relying on an automatic RTL conversion, you should now use `"start"` instead.
```